app/services/builders/chartkick/base.rb

```diff
@@ -23,6 +23,19 @@ def build_distribution_data(entities)
         end
         entities_by_interval.sort_by { |key, _| key.slice(/[0-9]*[+-]/).to_i }
       end
+
+      def build_success_rate(entities)
+        intervals = build_distribution_data(entities)
+        total = intervals.map { |interval| interval[1] }
+                         .reduce(0) { |sum, num| sum + num }
+        return if total.zero?
```

Comment:
will the rest of the code work if you return `nil` here? or will it fail with a nil exception when trying to access the data?

Reply:
When it returns `nil`, then it doesn't show the rate on the web. Take a look at `app/views/development_metrics/departments.erb`.